.eslintrc.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
     '@typescript-eslint/ban-ts-ignore': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     'react/react-in-jsx-scope': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
   },
```

Comment:
'@typescript&react': 'off', )))

Reply:
In css workshop we have inputs that should do nothing. Just showing us a color of cursor. But we have React warnings when we do not produce onChange function to input, so i set empty function there to mute it. And disable this rule because in future it can be the same cases.